### PR TITLE
Update examples and fix ParseAndVerify

### LINKED ISSUE #30: Where should I give the secret on validation?
In the validation example here https://github.com/cristalhq/jwt/blob/master/example_validate_test.go, it seems I didn't see any where to provide the secret/password on validation?

### PR DESCRIPTION
**Pull request by cristaloleg**
Fixes #30

### CODE REVIEW

**Comment by cristaloleg** [comment]
@elgs please, have a look 😄

File: parse.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func ParseAndVerify(raw []byte, signer Signer) (*Token, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err := signer.Verify(token.signature, token.payload); err != nil {
+	if err := signer.Verify(token.payload, token.signature); err != nil {
```
Comment (cristaloleg) [inline parse.go:69]: Ooops...

**Comment by elgs** [comment]
Thank you sooo much for the fast response!! I really appreciate!

**Comment by cristaloleg** [comment]
Feel free to update to the latest v1.2.0 https://github.com/cristalhq/jwt/releases/tag/v1.2.0

**Pull request merged**